SECP-service/src/main/resources/assets/app/directives/chat/chat-message.css

```diff
@@ -0,0 +1,2 @@
+/**** Chat message css *****/
+
```

Comment:
This will be used for chat specific css right?

Reply:
Yes. This would be the chat-message directive css. There is a lot of css in the chats.css class. Some of them needs to be moved here.

Reply:
👍🏻